doc/examples/filters/plot_deconvolution.py

```diff
@@ -19,23 +19,32 @@
 
 import numpy as np
 import matplotlib.pyplot as plt
+import skimage as ski
 
 from scipy.signal import convolve2d as conv2
 
-from skimage import color, data, restoration
 
 rng = np.random.default_rng()
 
-astro = color.rgb2gray(data.astronaut())
+# Convert astronaut image to grayscale
+astro = ski.color.rgb2gray(ski.data.astronaut())
 
+# Define PSF
 psf = np.ones((5, 5)) / 25
-astro = conv2(astro, psf, 'same')
-# Add Noise to Image
-astro_noisy = astro.copy()
-astro_noisy += (rng.poisson(lam=25, size=astro.shape) - 10) / 255.0
 
-# Restore Image using Richardson-Lucy algorithm
-deconvolved_RL = restoration.richardson_lucy(astro_noisy, psf, num_iter=30)
+# Convolve image with the PSF to simulate a blurred image
+astro_blurred = conv2(astro, psf, 'same')
+
+# Add Poisson noise to the blurred image (https://en.wikipedia.org/wiki/Shot_noise)
+max_photon_count = 1000
+astro_noisy = rng.poisson(astro_blurred * max_photon_count) / max_photon_count
+
+# Normalize noisy image
+# (for images of float dtype, skimage assumes their values range from -1 to 1 or 0 to 1)
```

Comment:
```suggestion
```

This will change soon, so let's leave it out.

Reply:
Even better if we could get away without normalizing at all; not sure if richardson_lucy supports that with a flag already.

Reply:
Well, it's the `clip=True` parameter:

https://github.com/scikit-image/scikit-image/blob/e8a42ba85aaf5fd9322ef9ca51bc21063b22fcae/skimage/restoration/deconvolution.py#L361-L378

I tried using `clip=False` (because we don't have pipeline issues/questions in this example) but then I suspect you need to adjust `num_iter`... I was getting a very dark (low contrast) image, even using the `vmin, vmax` parameters when plotting.